calico/felix/futils.py

```diff
@@ -630,3 +632,34 @@ def find_longest_prefix(strs):
             else:
                 longest_prefix = longest_prefix[:shared_len]
     return longest_prefix
+
+
+def usage_count(calico_version, hostname, cluster_guid, cluster_size, cluster_type):
```

Comment:
I dislike that the code is still using "usage_count" terminology when we agreed that it isn't a great name.  Why isn't this function called "report_usage" or "report_usage_and_get_warnings" or similar?